lib/jsonutil/jsonutil.go

```diff
@@ -0,0 +1,24 @@
+package jsonutil
+
+import (
+	"encoding/json"
+)
+
+// SanitizePayload will take in a JSON string, and return a JSON string that has been sanitized (removed duplicate keys)
+func SanitizePayload(val interface{}) (interface{}, error) {
+	var jsonMap map[string]interface{}
```

Comment:
Can move `var jsonMap map[string]interface{}` to after the `val.(string)` check.